app/forms/declaration/GoodsLocation.scala

```diff
@@ -57,13 +57,13 @@ object GoodsLocation extends DeclarationPage {
     "addressLine" -> optional(
       text().verifying("supplementary.goodsLocation.addressLine.error", isAlphanumericWithAllowedSpecialCharacters and noLongerThan(70))
     ),
-    "postCode" -> optional(
-      text().verifying("supplementary.goodsLocation.postCode.error", isAlphanumericWithAllowedSpecialCharacters and noLongerThan(9))
-    ),
     "city" -> optional(
       text()
         .verifying("supplementary.goodsLocation.city.error", isAlphanumericWithAllowedSpecialCharacters and noLongerThan(35))
     ),
+    "postCode" -> optional(
+      text().verifying("supplementary.goodsLocation.postCode.error", isAlphanumericWithAllowedSpecialCharacters and noLongerThan(9))
+    ),
     "country" ->
```

Comment:
This "postCode" is just a name to reuse it in forms. If you changed the order I think that now `postCode` will be stored in the `city` place. At the end it's just `GoodsLocation.apply`.

Reply:
We checked the CYA screen and came back from there. PostCode & city are getting stored in respective places only.

Reply:
But DB suggest you are right. We will change the constructor too now.

Reply:
Yes, the UI will work similar, because you're saving and reading from the same wrong place, but the declaration and model in database will be wrong

Reply:
Fixed

Reply:
Please review and approve